### PR TITLE
(docs) update docs/mypy_plugin.md

### PR DESCRIPTION
**Pull request by vladdoster**
## Change Summary

- remove `just` which adds verbiage
- add `the` to correct article agreement

## Related issue number

N/A

## Checklist

* [x] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [ ] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

### CODE REVIEW

File: docs/mypy_plugin.md [hunk 1]
```diff
@@ -76,11 +76,11 @@ There are other benefits too! See below for more details.
 
 ### Enabling the Plugin
 
-To enable the plugin, just add `pydantic.mypy` to the list of plugins in your
+To enable the plugin, add `pydantic.mypy` to the list of plugins in your
 [mypy config file](https://mypy.readthedocs.io/en/latest/config_file.html)
 (this could be `mypy.ini` or `setup.cfg`).
 
-To get started, all you need to do is create a `mypy.ini` file with following contents:
+To get started, all you need to do is create a `mypy.ini` file with the following contents:
```
Comment (PrettyWood) [inline docs/mypy_plugin.md:83]:
```suggestion
To get started, all you need to do is create a `mypy.ini` file with the following content:
```

**Review by PrettyWood — COMMENTED** [review]
You can switch from draft to ready and I'll merge thanks!

**Comment by vladdoster** [comment]
oof, just saw this :/

**Comment by samuelcolvin** [comment]
Minimal change and the fork has been deleted so I can't update this.

**Pull request closed without merging**